packages/block-editor/src/components/media-flow/style.scss

```diff
@@ -0,0 +1,43 @@
+.components-popover:not(.is-mobile).is-bottom {
+	z-index: 1000000 !important;
+}
+
```

Comment:
@jorgefilipecosta I had to add this above because otherwise the url input's suggestion box would appear below the menu if the menu was shown on top. This needs to override a rule in _z-index.scss:

```
// Shows adminbar quicklink submenu above bottom popover:
// #wpadminbar ul li {z-index: 99999;}
".components-popover:not(.is-mobile).is-bottom": 99990,
```

Do you know what is the problem if `.components-popover.is-bottom` has a z-index of `1000000`?

Reply:
after giving a look at the file, I see no problem with the value but I still think it shouldn't be hardcoded into the file directly, it should be declared in  `z-index.scss` and called using `z-index(".classname")`